static/scripts/secret.ts

```diff
@@ -35,18 +35,19 @@ interface ConfLabel {
 }
 
 interface IConf {
-  PSK?: string;
-  baseMultiplier?: number;
-  timeLabels?: ConfLabel[];
-  priorityLabels?: ConfLabel[];
-  autoPayMode?: boolean;
-  analyticsMode?: boolean;
+  'PSK'?: string;
```

Comment:
What is PSK?

Reply:
private-key-key-name

Reply:
Perhaps we should rename this to something clear, like `private-key-name` ?

Reply:
yes, `private-key` seems to be more expressive

if `private-key` is fine I can update the bot's code and the config generation page

Reply:
more likely PSK = Privately Shared Key

Reply:
@rndquu you can make the decision and update the config generation.

Reply:
> @rndquu you can make the decision and update the config generation.

ok